tools/cloud_functions/gcs_event_based_ingest/gcs_ocn_bq_ingest/monitoring/gcf_ingest_latest_by_table.sql

```diff
@@ -0,0 +1,33 @@
+CREATE OR REPLACE VIEW
+  `default-dataset`.gcf_ingest_latest_by_table AS
```

Comment:
Let's use a better name than default-dataset.
I don't think there can be `-` in dataset names FYI https://cloud.google.com/bigquery/docs/datasets#dataset-naming
Perhaps `gcf_ingest_monitoring_views`